cli/crates/cli/src/cli_input.rs

```diff
@@ -51,7 +51,17 @@ pub fn build_cli() -> Command {
         "}))
         .subcommand(Command::new("login").about("Logs into your Grafbase account"))
         .subcommand(Command::new("logout").about("Logs out of your Grafbase account"))
-        .subcommand(Command::new("create").about("Set up and deploy a new project"))
+        .subcommand(
+            Command::new("create").about("Set up and deploy a new project").args(&[
```

Comment:
Not suggesting we change it, but I've been wondering for a while: is there a reason we're defining all this by hand rather than using the derives in clap?

Reply:
I recall the derive API being limited in comparison, we could look into it again if it has parity